Source/WebCore/platform/graphics/GraphicsLayer.cpp

```diff
@@ -421,6 +422,24 @@ void GraphicsLayer::setMaskLayer(RefPtr<GraphicsLayer>&& layer)
     m_maskLayer = WTFMove(layer);
 }
 
+MediaPlayerVideoGravity GraphicsLayer::videoGravity() const
+{
+#if USE(CA)
+    return m_videoGravity;
+#else
+    return MediaPlayerVideoGravity::ResizeAspect;
```

Comment:
`MediaPlayerVideoGravity::ResizeAspect` is the hard coded default gravity in a bunch of methods now so it might be worth defining a named constant.

Reply:
One problem is that `ResizeAspect != 0`. If we are going to make a default value, we should also have it be equivalent to 0.